### PR TITLE
Made urlBarIconContainer clickable to display connection info

### LINKED ISSUE #7888: Connection info click area is on the urlbarIcon, should be on urlbarIconContainer
## Test plan
See https://github.com/brave/browser-laptop/pull/13147

## Original issue description
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
The click area is pretty small limited only to the urlbaricon, even a little off causes the click to be not captured. The click should be on the urlbarIconContainer which increases the hit area 

- Platform (Win7, 8, 10? macOS? Linux distro?):
All

- Brave Version (revision SHA):
Brave	0.14.0
rev	ffa6f8f

- Steps to reproduce:
    1. Open any http/https site
    2. Click on the padlock icon the connection info is shown 
    3. Move cursor sightly off the padlock but visually the pointer still looks on the padlock, click not recognized and connection info is not shown 

  **All other browser has the large click area to show the connection info window/modal**
    
- Actual result:
Connection info click area is on the urlbarIcon, should be on urlbarIconContainer

- Expected result:
Connection info not shown if the click is off by a marginal amount

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
What it is now
image
What it should be 
image

- Any related issues: #6704
cc: @bradleyrichter @bsclifton

### PR DESCRIPTION
**Pull request by srirambv**
Fixes #7888 

Redo of https://github.com/brave/browser-laptop/pull/13147 (which unintentionally introduced #13163)
- original commit was reverted in master with https://github.com/brave/browser-laptop/commit/49a9b7379bbaf03d48bb1438512195fd6b3fdb66
- commit re-added here (first commit in this PR)
- second commit fixes the issue discovered

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Test Plan:
1. Visit https;//github.com, connection info icon should be grey and EVC should be in green
2. Visit http;//pdf995.com, connection info icon should be red
3. Visit https://https-everywhere.badssl.com, connection info icon should be gery
4. Visit https://mixed-script.badssl.com, connection icon should be grey, click and load unsafe script, connection info icon should be red

connection1

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by bsclifton** [comment]
@MargarytaChepiga can you please review this PR? @srirambv had some tweaks to your original contribution 😄 BTW- you should have an invitation to be an official collaborator now.  Once you accept, you can assign yourself to issues 😄

**Comment by bsclifton** [comment]
@MargarytaChepiga (and everybody else) - apologies for what happened here. I didn't do a proper review when accepting the original pull request (definitely my bad). Part of a good review includes checking the Travis CI status, which I neglected to do ☹️ 

For sensitive areas like the URL bar, it's also great to loop in security.  To remedy this, I rolled back the code in master... re-introduced it here... and then created a security review 😄 
https://github.com/brave/internal/issues/210

**Comment by diracdeltas** [comment]
nit: this introduces more white space next to the url bar icon than there was previously
screen shot 2018-02-16 at 6 03 06 pm

before:
screen shot 2018-02-16 at 6 02 59 pm

otherwise lgtm!! thanks for addressing my concerns. :)

File: less/navigationBar.less [hunk 1]
```diff
@@ -835,7 +832,7 @@
         background-position: center;
         position: relative;
         bottom: -1px;
-        margin-left: 3px;
+        padding: 5px 7px 5px 7px;
```
Comment (MargarytaChepiga) [inline less/navigationBar.less]: The extra space between an icon and address is caused by the padding. Looked okay with the secure connection, however as @diracdeltas mentioned with others not really. Changing to 5px 5px 5px 5px makes it better.

**Comment by diracdeltas** [comment]
@MargarytaChepiga yes, please feel free to make that change! to have it be part of this PR, either:

1. @srirambv can give you permission to push to this branch on his fork:  https://github.com/srirambv/browser-laptop.git 

or

2. You can push the change to your fork and he can merge/cherry-pick it into this branch.

**Comment by srirambv** [comment]
@MargarytaChepiga  please go ahead and make the changes. I've added you to the branch

**Comment by MargarytaChepiga** [comment]
Sorry for the delay. I'm facing issues pushing the changes. Probably I am doing something wrong. 
I added  https://github.com/srirambv/browser-laptop.git as a remote, then I did git fetch, then checked out to the connection branch where I've made the changes, tested them, commited and when I try to push I get the following:
accessdenied
And just in case adding this:
remote

**Comment by srirambv** [comment]
@MargarytaChepiga  I think its due to the fact that its still not given access. 
image

Here's the link https://github.com/srirambv/browser-laptop/invitations . Could you please try once again

**Comment by MargarytaChepiga** [comment]
@srirambv My bad, thank you so much!

**Comment by diracdeltas** [comment]
Thanks, this looks great. However it doesn't seem to fix https://github.com/brave/browser-laptop/issues/13011 for me, not sure why. When I click on the EV certificate info on github.com, nothing happens unless I click on the lock icon.

**Comment by MargarytaChepiga** [comment]
@diracdeltas That's weird... I just checked it again, just to be sure. It seems to work fine.
test

**Pull request merged**